scss/elements/_lists.scss

```diff
@@ -19,7 +19,9 @@
 ul,
 ol {
   list-style-type: none;
-  margin: 0;
+  margin-top: 0;
```

Comment:
Properties should be ordered list-style-type, margin-bottom, margin-left, margin-top, padding